stonk_market_mayhem2.py

```
@@ -0,0 +1,76 @@
+    cost = stocks[stock] * qty
+    if money >= cost:
+        money -= cost
+        if stock in portfolio:
+            portfolio[stock] += qty
+        else:
+            portfolio[stock] = qty
+        print(f"Bought {qty} shares of {stock}")
+    else:
+        print("You broke.")
+
+def sell_stock():
```

Ruff (0.8.2)

Missing return type annotation for public function `sell_stock`

Add return type annotation: `None`

(ANN201)